server/routes/parcelActions.js

```diff
@@ -1,7 +1,19 @@
 const validationSchema = require('../schema/parcelActions')
 const parcelActionsService = require('../services/parcelActionsService')
+const parcelService = require('../services/parcelService')
+const actionsService = require('../services/actionsService')
+const rulesEngineHelper = require('../rules-engine/helper')
 
-module.exports = {
+const quantityBoundsResponseBuilder = (action, runResult) => ({
+  id: action.id,
+  description: action.description,
+  input: {
+    ...action.input,
+    upperbound: runResult.upperbound && Math.round(runResult.upperbound.value * 100) / 100
```

Comment:
This isn't a 100% reliable way to get 2 decimal places. For example, 1.005 => 1, rather than 1.01.

A solution is apparently to add the smallest possible number (`Number.EPSILON`) to the upperbound before multiplying by 100:
```
console.log(Math.round((runResult.upperbound.value + Number.EPSILON) * 100) / 100)
```
Adapted from https://stackoverflow.com/a/11832950

Reply:
Fair point, I googled a solution rather than actually thinking about it myself, but I did wonder if something like this might happen. I'll add your change.